### PR TITLE
MesherySettingsPerformanceComponent Converted class-based component to functional component 

### LINKED ISSUE #8813: [UI] Convert `MesherySettingsPerformanceComponent` class-based component to functional component
### Current Behavior
`MesherySettingsPerformanceComponent` is a  class-based component

### Desired Behavior
`MesherySettingsPerformanceComponent` should be a functional component

### Implementation
Convert all the Class-based components to new updated functional components. It is important here to convert a class-based component to a functional component you may need to convert some functions as well. Check if the function is used in other components or not.

Also, make sure to use the hooks while converting them. For example, you need to covert withNotify with useNotification, useRouter hook. Please check if a hook for functional components is available and use that.

Here is a quick blog that will help you get started with : [Link](https://www.digitalocean.com/community/tutorials/five-ways-to-convert-react-class-components-to-functional-components-with-react-hooks)

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by csengineer1990**
Converted MesherySettingsPerformanceComponent.js  class-based component to functional component 

**Notes for Reviewers**
Please check the above changes if I did anything wrong please let me know.

This PR fixes #8813

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by saurabh100ni** [comment]
Hey @csengineer1990 thanks for raising PR,
Let's discuss this in the Meshery Development call today,
Please add this in Doc as your agenda item (if it isn't already there): https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit

File: ui/components/MesherySettingsPerformanceComponent.js [hunk 1]
```diff
@@ -285,8 +287,260 @@ class MesherySettingsPerformanceComponent extends React.Component {
       </NoSsr>
     );
   }
-}
+}*/
+
+function MesherySettingsPerformanceComponent(props) {
+  const { classes, qps: initialQPS, c: initialC, t: initialT, gen: initialGen, notify } = props;
```
Comment (Yashsharma1911) [inline ui/components/MesherySettingsPerformanceComponent.js:293]: I think these names `c` and `t` aren't very descriptive can we make them more descriptive?

Comment (csengineer1990) [reply ui/components/MesherySettingsPerformanceComponent.js:293]: yes sir we can make names descriptive.

Comment (Yashsharma1911) [reply ui/components/MesherySettingsPerformanceComponent.js:293]: Thankyou

**Comment by l5io** [comment]
🚀 Preview for commit 80423150dbc1cc2b79bfb961bbb542e20c3ad587 at: https://6512c1b024fcb102a0f3f950--meshery-docs-preview.netlify.app

File: ui/components/MesherySettingsPerformanceComponent.js (hunk 1 above)
Comment (csengineer1990) [reply ui/components/MesherySettingsPerformanceComponent.js]: sir, please check I have updated the correction. If there is anything wrong please let me know and guide me.

Comment (csengineer1990) [reply ui/components/MesherySettingsPerformanceComponent.js]: sir, please check I have updated the correction. If there is anything wrong please let me know and guide me.

**Comment by l5io** [comment]
🚀 Preview for commit 79f8bccaca52365a8aae89ab6d0a5128511d9e1f at: https://651570128f5f5916116f8751--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9b8dd292f38c737e09507288fd992ae75cedae7b at: https://65157a508f5f591e7a6f913d--meshery-docs-preview.netlify.app

**Comment by csengineer1990** [comment]
While fixing the conflict issue I synced the repo in GitHub so it closed the pull request. Now I made a new pull request ( #8970)  please check that with the above corrections.

**Pull request closed without merging**